modules/menuClass.js

```diff
@@ -0,0 +1,145 @@
+import {
+  blockDisplay, formSection, contactSection, bookTitle, bookAuthor, form, bookList,
+} from './variables.js';
+
+export const listBtn = document.querySelector('.list-btn');
+export const addBtn1 = document.querySelector('.adde-btn-1');
+export const contactBtn = document.querySelector('.contact-btn');
+export const displayList = () => {
+  blockDisplay.style.display = 'flex';
+  formSection.style.display = 'none';
+  contactSection.style.display = 'none';
+};
+
+export const displayBookForm = () => {
+  blockDisplay.style.display = 'none';
+  formSection.style.display = 'flex';
+  contactSection.style.display = 'none';
+};
+
+export const displayContact = () => {
+  contactSection.style.display = 'flex';
+  blockDisplay.style.display = 'none';
+  formSection.style.display = 'none';
+};
+
+/* eslint-disable keyword-spacing */
+
```

Comment:
- [ ] (Required) Kindly remove this `eslint-disable` comment. `Eslint` helps you to write clean JavaScript codes in a structured way. Using the `eslint-disable` line is considered bad practice in JavaScript. Kindly remove it and investigate the error.